docs/examples/zoom.ipynb

```diff
@@ -0,0 +1,285 @@
+    ")\n",
+    "\n",
+    "params[QBins] = sc.geomspace(dim='Q', start=0.004, stop=0.8, num=141, unit='1/angstrom')\n",
+    "params[NonBackgroundWavelengthRange] = sc.array(\n",
+    "    dims=['wavelength'], values=[0.7, 17.1], unit='angstrom'\n",
+    ")\n",
+    "params[CorrectForGravity] = True\n",
+    "params[UncertaintyBroadcastMode] = UncertaintyBroadcastMode.upper_bound"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "0b47024f",
+   "metadata": {},
+   "source": [
+    "### Setup reduction pipeline"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "fdcaec37",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "providers = sans.providers + sans.isis.providers\n",
+    "providers = providers + (\n",
+    "    sans.isis.transmission_from_background_run,\n",
+    "    sans.isis.transmission_from_sample_run,\n",
+    ")\n",
+    "pipeline = sciline.Pipeline(providers, params=params)\n",
+    "pipeline.set_param_table(masks)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "951aec10",
+   "metadata": {},
+   "source": [
+    "If Mantid is available, we can use it to load data files.\n",
+    "**You must configure the `DataFolder` below to point to the directory containing the data files.**\n",
+    "Otherwise, we fall back to load intermediate data files that have been prepared for the concrete example in this notebook.\n",
+    "If you want to use the workflow with different files you must have Mantid installed:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "66237b3e",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "try:\n",
+    "    from mantid import ConfigService\n",
+    "    import esssans.isis.mantidio\n",
+    "\n",
+    "    cfg = ConfigService.Instance()\n",
+    "    cfg.setLogLevel(3)  # Silence verbose load via Mantid\n",
+    "\n",
+    "    pipeline[sans.isis.DataFolder] = 'zoom_data'\n",
+    "    for provider in sans.isis.mantidio.providers:\n",
+    "        pipeline.insert(provider)\n",
+    "except ImportError:\n",
+    "    import esssans.isis.data\n",
+    "\n",
+    "    for provider in sans.isis.data.providers:\n",
+    "        pipeline.insert(provider)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "703ffc1e",
+   "metadata": {},
+   "source": [
+    "## Reduction\n",
+    "\n",
+    "### The reduction workflow"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "cae69232",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "iofq = pipeline.get(IofQ[SampleRun])\n",
+    "iofq.visualize(graph_attr={'rankdir': 'LR'}, compact=True)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "77687728",
+   "metadata": {},
+   "source": [
+    "### Running the workflow"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "558327ff",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "da = iofq.compute()\n",
```

Comment:
Maybe we should silence the `pooch` warnings everywhere?
![Screenshot at 2024-01-18 14-17-56](https://github.com/scipp/esssans/assets/39047984/40d9186e-f21e-4cd2-ad79-ccedead62456)